pytorch_forecasting/models/base_model.py

```diff
@@ -18,7 +18,7 @@
 from lightning.pytorch.utilities.parsing import AttributeDict, get_init_args
 import matplotlib.pyplot as plt
 import numpy as np
-from numpy.lib.function_base import iterable
+from numpy import iterable
```

Comment:
no request, just making a note: *this is fine* wrt downwards compatibility because this is also an alternative import location for `numpy 1`.